templates/19.0/odoo.cfg.tmpl

```diff
@@ -5,8 +5,8 @@ auto_reload = False
 db_template = ${DB_TEMPLATE}
 db_host = ${DB_HOST}
 db_port = ${DB_PORT}
-db_replica_host = ${DB_REPLICA_HOST}
-db_replica_port = ${DB_REPLICA_PORT}
+db_replica_host = {{ DB_REPLICA_HOST if DB_REPLICA_HOST is defined and DB_REPLICA_HOST else "" }}
+db_replica_port = {{ DB_REPLICA_PORT if DB_REPLICA_PORT is defined and DB_REPLICA_PORT else "" }}
```

Comment:
@acsonefho let's do this in `answers.sh` so everything is in one place.

Reply:
And they need to be set to `None` instead of `False`. We can see that by running `ODOO_RC=/tmp/odoo.cfg odoo -s` to see all the default values.